content/en/plugins/github-api/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: GiHub API Plugin for Spinnaker
+linkTitle: GitHub API
+no_list: true
+description: >
+  Armory's GitHub API plugin for Spinnaker streamlines integration with GitHub Actions, filling the native support gap. The plugin enables easy triggering of GitHub workflows, dynamic control of Spinnaker pipelines based on workflow outcomes, and seamless synchronization of GitHub Deployment statuses with Spinnaker pipeline conclusions.
+---
+
+![Proprietary](/images/proprietary.svg) ![Early Access](/images/ea.svg)
+
+## GitHub API Plugin features
+
+The GitHub API Plugin provides the following GitHub integration features:
+
+- Trigger a Spinnaker pipeline from a GitHub workflow using **workflow_dispatch** or **repo_dispatch** events
+- Trigger a Spinnaker pipeline automatically when a GitHub workflow finishes successfully
+- Trigger a Spinnaker pipeline when GitHub creates a new GitHub Deployment
+- Monitor a GitHub workflow and finish pipeline execution based on the GitHub workflow result
+- Update GitHub deployment status based on Spinnaker pipeline outcome
+- View Github Action Logs in Spinnaker -- there is no need to navigate to GitHub to view the logs
+
+The GitHub API plugin uses [GitHub Apps](https://docs.github.com/en/apps/overview) to integrate with GitHub. GitHup Apps provide webhooks and narrow, specific permissions. You install a GitHub App in your organization and specify which repositories the GitHub API plugin can access. 
+
+## How the plugin works with GitHub
+
+```mermaid
+graph TD
+	Orca -->|Workflow Trigger| Igor
+	Gate --> |Trigger based on Deployment event|Echo
+	Igor -->|Trigger/Monitor triggered workflows| Github
+	Echo -->|Pipeline triggers|Orca
+	Orca -->|Pipeline notifications|Echo
+	Github --> |Deployment event| Gate
+	Echo --> |Notifications|Github
+```
+
+For example, the plugin processes a GitHub Deployment event like this:
+
+```mermaid
+sequenceDiagram
+participant ghActions as "Github Workflow job"
+participant gh as "Github"
+participant gate as "Gate"
+participant echo as "Echo"
+participant orca as "Orca"
+
+ghActions ->> gh: "Create Deployment"
+gh ->> ghActions: "Deployment created"
+gh ->> gate: "Deployment Event"
+gate ->> echo: "Deployment trigger event"
+echo ->> orca: "Trigger pipelines matching the Trigger event"
+orca ->> gh: "Deployment status update: in_progress, success, failure, or error"
+```
+
+## Compatibility matrix
```

Comment:
We will need to add a Known Issues section. For example we know that this plugin conflicts with the Armory.Header plugin's default version installed in 2.30.x. That plugin needs to be updated to 0.2.0 in order for the UI to function correctly. Maybe this needs to be added in the release section?

Reply:
@christosarvanitis Thanks for letting me know. I'll add these to the release-notes.md file.